sdk-libs/halo-notifications/src/main/java/com/mobgen/halo/android/notifications/HaloNotificationsApi.java

```diff
@@ -113,6 +115,8 @@ public void onEventReceived(@NonNull Event event) {
                 updateToken();
             }
         });
+
+        NotificationService.setIdGenerator(new HaloNotificationIdGenerator());
```

Comment:
I would add this as part of the service initialization more than an API call. What do you think?